<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MuniEntry</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>741</width>
      <height>171</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Criminal</string>
     </attribute>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="4">
       <widget class="QPushButton" name="pushButton_2">
        <property name="text">
         <string>PushButton</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="4">
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="MotionEntryDialog">
        <property name="text">
         <string>Motion Entry</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="3" rowspan="3">
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="1" rowspan="3">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="0">
       <widget class="QPushButton" name="TransferEntryDialog">
        <property name="text">
         <string>Transfer Entry</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="YellowFormDialog">
        <property name="text">
         <string>Pending CrTr Entry (Yellow Form)</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Civil</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Jury</string>
     </attribute>
     <widget class="QPushButton" name="JuryInstructionsDialog">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>179</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Jury Instructions</string>
      </property>
     </widget>
     <widget class="QPushButton" name="VerdictFormDialog">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>179</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Verdict Form</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionOptions"/>
    <addaction name="actionExit"/>
    <addaction name="separator"/>
    <addaction name="actionExit_2"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionOptions">
   <property name="text">
    <string>Options</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Templates</string>
   </property>
  </action>
  <action name="actionExit_2">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionGeneral_Motion_Entry">
   <property name="text">
    <string>General Motion Entry</string>
   </property>
  </action>
  <action name="actionFail_to_Appear_Minor_Traffic_Offense">
   <property name="text">
    <string>Fail to Appear - Minor Traffic Offense</string>
   </property>
  </action>
  <action name="actionOmnibus_Motion_Form">
   <property name="text">
    <string>Omnibus Motion Form</string>
   </property>
  </action>
  <action name="actionJury_Instructions">
   <property name="text">
    <string>Jury Instructions</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>MotionEntryDialog</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>pushButtonDialog()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>81</x>
     <y>178</y>
    </hint>
    <hint type="destinationlabel">
     <x>143</x>
     <y>262</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>TransferEntryDialog</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>pushButtonDialog()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>73</x>
     <y>86</y>
    </hint>
    <hint type="destinationlabel">
     <x>121</x>
     <y>26</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>YellowFormDialog</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>pushButtonDialog()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>402</x>
     <y>88</y>
    </hint>
    <hint type="destinationlabel">
     <x>447</x>
     <y>265</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>pushButtonJuryInstructions()</slot>
  <slot>pushButtonTransferEntry()</slot>
  <slot>pushButtonVerdictForm()</slot>
  <slot>pushButtonOmnibusMotion()</slot>
  <slot>pushButtonDialog()</slot>
 </slots>
</ui>
